core/models/Auth.js

```diff
@@ -13,6 +13,7 @@ module.exports = {
     ],
     "auth/me": [
       PolicyService.checkAndInjectOperator,
+      PolicyService.hasRight("auth", "read"),
```

Comment:
I don't think it's a good idea. Everyone should know whether he is logged in or not. And, I don't think we need matrix rights on this controller – but this is more client problem, we'll simply not display this model at the rights editor.